test/appium/tests/base_test_case.py

```diff
@@ -247,11 +246,12 @@ def teardown_class(cls):
         cls.loop.close()
 
 
-environment = LocalMultipleDeviceTestCase if pytest.config.getoption('env') == 'local' else SauceMultipleDeviceTestCase
-
-
-class MultipleDeviceTestCase(environment):
-    pass
+if pytest.config.getoption('env') == 'local':
+    class MultipleDeviceTestCase(LocalMultipleDeviceTestCase):
```

Comment:
you can do `MultipleDeviceTestCase = LocalMultipleDeviceTestCase`

Reply:
Agree, makes sense here, thanks @antdanchenko